### PR TITLE
Climber Commands

### PR DESCRIPTION
**Pull request by jjieshenwang**
Commands for the climber( climberunwind, climberextend, climber retract, climberclimb, and climberstop)

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 1]
```diff
@@ -1,33 +1,65 @@
...
- * Authors:
+ /* ==================================================
+ * Authors: Jason and Alexandra
  *
  * --------------------------------------------------
- * Description:
+ * Description: 
+ * Contains the functions to control the climber.
  *
  * ================================================== */
```
Comment (bricktrooper) [inline src/main/java/frc/robot/subsystems/Climber.java]: Could you do a merge from `master` into your branch `climber` and push that?  For some reason it thinks that you are adding this code again even though it was adready added in the last PR.  First switch your branch to `master`.  Then pull `master`.  Finally merge `master` into `climber`.  Then switch back to `climber` and commit the merge and push the changes.

Generally you either want to make a new branch for every PR, or always make sure to merge from `master` first.  The most important thing to remember is to pull from `master` before making a new branch or updating an old branch and before changing any code.

**Review by bricktrooper — COMMENTED** [review]
Looks good, just some git technicalities.

**Review by bricktrooper — COMMENTED** [review]
Can you set the default command for climber as `ClimberStop()` inside `Robot.java`?

**Review by bricktrooper — APPROVED** [review]

**Comment by jjieshenwang** [comment]
Commands for the climber(unwind, extend, retract, climb, and stop)

**Pull request merged**